tests/x11/updates_packagekit_gpk.pm

```diff
@@ -41,8 +41,12 @@ sub run() {
             send_key "alt-i";    # install
 
             # Authenticate on SLES
-            if (check_screen("updates_authenticate")) {
+            # FIXME: actually only do that on SLE
```

Comment:
I don't understand the FIXME. I think your implementation with the "match_has_tag" is just fine